// Copyright 2024 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// If you change or add any fields in this file, update proto_visitors.h and
// potentially proto_enum_conversions.{h, cc}.

syntax = "proto2";

option java_multiple_files = true;
option java_package = "org.chromium.components.sync.protocol";

option optimize_for = LITE_RUNTIME;

package sync_pb;

message PlusAddressSpecifics {
  // A stable, unique identifier, generated by the PlusAddress backend.
  optional string profile_id = 1;
  // The origin that the `plus_email` is generated for.
  optional string facet = 2;
  optional PlusEmail plus_email = 3;
}

message PlusEmail {
  optional string email_address = 1;
}
